resources/test_servers.yml

```diff
@@ -1,27 +1,34 @@
 version: '2'
 services:
   zookeeper:
-    image: wurstmeister/zookeeper
+    image: hub.docker.prod.walmart.com/wurstmeister/zookeeper
```

Comment:
You want yours here or public one?

Reply:
Sorry, I missed this one and will put it back to public.

Reply:
changed, commited, and squashed

Reply:
Missed this.
reset back to public docker hub
commited and squashed